cost/aws/idle_compute_instances/idle_compute_instances.pt

```diff
@@ -687,8 +689,13 @@ script "js_instance_cost_mapping", type:"javascript" do
         })
       }
     })
-    total_savings=cur + ' '+formatNumber(total_savings.toFixed(3), separator);
-    message="The total estimated monthly savings are "+total_savings;
+    if(count){
+      total_savings=cur + ' '+formatNumber(total_savings.toFixed(3), separator);
```

Comment:
I noticed you changed the occurrence of this in the above policy to use `Math.round()` instead, what was the reasoning for that? And why was that not needed here? Or was it just missed here?

Reply:
There are few policies for which cost data is very low and is coming as 0.00052346, so we discussed with Curt to use toFixed(3) in those policies as Math.round() will round them to 0. The other policies were having higher cost data so we reverted the toFixed(3). In this policy, we are receiving as $0 from Optima, so we have left the toFixed(3) in this policy. Thanks